### ISSUE TITLE
Absence of examples of some methods

### ISSUE THREAD

**Issue by Engineer1999** [arduino-libraries/Servo#51]
While browsing through the codebase, I came to know that examples of some functionality in the Servo library are not available. Below are the methods for which examples are needed.
-> servo.writeMicroseconds()
-> servo.read()
-> servo.attached()
-> servo.detach()
